site/content/en/docs/build-clusters.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "GKE Build Clusters"
```

Comment:
Note that for prow, a build cluster could be any k8s cluster, could be GKE, AKS, or EKS. I think it's fine to only document GKE build cluster for now. But should also keep in mind that build cluster from other providers might also be documented by someone else.

Reply:
Exactly.